### PR TITLE
Add postgresql restore doc

### PR DESCRIPTION
**Pull request by zugao**
## Summary

Documentation on how to restore a VSHN PostgreSQL database after data loss.

## Checklist

- [x] Try to isolate changes into separate PRs (to build a better changelog).
- [x] Categorize the PR by setting a good title and adding one of the labels:
      `change`, `decision`, `requirement/quality`, `requirement/functional`, `dependency`
      as they show up in the changelog

### CODE REVIEW

File: docs/modules/ROOT/pages/how-tos/appcat/vshn/postgres/manual-restore.adoc [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+----
+kubectl get XObjectBucket --as cluster-admin <name>
+----
+<7> THe bucket name
+<8> S3 Cloud provider endpoint
+<9> Region of the bucket
+
+== Step 3 Restore the Database
+The restore process is straightforward. The process is to tell to the PostgreSQL cluster to use our backup.
+Update `SGCluster` resource with the following snippet of code:
```
Comment (Kidswiss) [inline docs/modules/ROOT/pages/how-tos/appcat/vshn/postgres/manual-restore.adoc:153]: We'd have to re-create the object though, the `initialData` field only triggers on the very first startup of the cluster AFAIK.

File: docs/modules/ROOT/pages/how-tos/appcat/vshn/postgres/manual-restore.adoc [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+kubectl get XVSHNPostgreSQL <name-from-alert> -o=jsonpath='{.status.instanceNamespace}' --as cluster-admin
+----
+
+[WARNING]
+If the namespace does not exist then the database has been removed together with the PVCs
+In this case the data is lost and must be retrieved immediately from the S3 buckets.
+
+== Step 2 Prepare for Restore Operation
+Before restoring any data the database must be operational either with the old configuration or via a new setup.
+In case the XRD with the claim have been removed, the customer might need to provide the details of their Claim in order to
```
[the remark this answers is not in the document]
Comment (zugao) [reply docs/modules/ROOT/pages/how-tos/appcat/vshn/postgres/manual-restore.adoc:39]: There might be some edge cases where we still have the buckets no?

File: docs/modules/ROOT/pages/how-tos/appcat/vshn/postgres/manual-restore.adoc (hunk 1 above)
Comment (zugao) [reply docs/modules/ROOT/pages/how-tos/appcat/vshn/postgres/manual-restore.adoc:153]: True, we have to recreate the object. I will amend the changes.

File: docs/modules/ROOT/pages/how-tos/appcat/vshn/postgres/manual-restore.adoc (hunk 2 above)
[the remark this answers is not in the document]
Comment (zugao) [reply docs/modules/ROOT/pages/how-tos/appcat/vshn/postgres/manual-restore.adoc]: I updated the doc

[the remark this answers is not in the document]
Comment (Kidswiss) [reply docs/modules/ROOT/pages/how-tos/appcat/vshn/postgres/manual-restore.adoc]: There's a way to have deletion protection for buckets directly via the providers. However I'm not sure all our s3 providers have this implemented.

Also this provider-side-deletion protection is not exposed to the AppCat APIs. And I wouldn't actually do it, as layering so many protection mechanisms is at one point more harmful than helpful.

We probably need to adjust some things here anyway once we've implemented our webhook based protections.

[the remark this answers is not in the document]
Comment (zugao) [reply docs/modules/ROOT/pages/how-tos/appcat/vshn/postgres/manual-restore.adoc]: I will remove this line from the docs then.

**Review by Kidswiss — APPROVED** [review]

**Pull request merged**